packages/cli/src/lib/compose/client.ts

```diff
@@ -54,7 +50,9 @@ export type ComposeClient = ReturnType<typeof composeClient>
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 type ParametersExceptFirst<F> = F extends (arg0: any, ...rest: infer R) => any ? R : never;
 
-export const localComposeClient = (composeFiles: string[] | Buffer, projectName?: string) => {
+type LocalComposeClientArgs = {composeFiles: string[] | Buffer; projectName?: string; env?: NodeJS.ProcessEnv}
```

Comment:
nit: the existing convention in the code base is that a separate type is not defined unless it's used somewhere outside of the func definition